src/pages/style.css

```diff
@@ -211,3 +211,35 @@ and negative margin.
     outline: none;
   }
 }
+
+/*
+***
+* Scroll offset global helper, which accounts for fixed headings
+* TODO: use this for docs and learn content as well. Will likely make sense
+* to make this shift when we move to shared MDX content components for
+* heading elements across both Docs and Learn.
+* CSS → MDX task: https://app.asana.com/0/1201987349274776/1201653123138729/f
+* "Reconcile" task: https://app.asana.com/0/1201987349274776/1202052090505082/f
+***
+*/
+.g-offset-scroll-margin-top {
```

Comment:
The styles under this class are duplicative of [similar styles in `dev-dot-content.module.css`](https://github.com/hashicorp/dev-portal/blob/8a4e6f77c77227c298f812deb0ccd6f7cce30229/src/components/dev-dot-content/dev-dot-content.module.css#L48).

I've intentionally chosen not to try to reconcile the `dev-dot-content.module.css` styles at this time, as it would significantly balloon the scope of this PR (we'd need to move to using MDX components for headings, and have those MDX components compose this class). Intent is to handle that work in a 🎟️ [Convert global-ish docs content CSS to MDX custom components](https://app.asana.com/0/1201987349274776/1201653123138729/f)

Reply:
I'm okay with this given your comments there / tickets! 🆗